File: official/docs/csharp/v4/batches/scan-forms.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using Newtonsoft.Json;
using EasyPost;
using EasyPost.Models.API;

namespace EasyPostExamples
{
    public class Examples
    {
        public static async Task Main()
        {
            var client = new EasyPost.Client("EASYPOST_API_KEY");

            Batch batch = await client.Batch.Retrieve("batch_...");

            await batch.GenerateScanForm();

            Console.WriteLine(JsonConvert.SerializeObject(batch, Formatting.Indented));
        }
    }
}

File: official/docs/golang/v4/batches/scan-forms.go
package example

import (
	"fmt"

	"github.com/EasyPost/easypost-go/v4"
)

func scanForms() {
	client := easypost.New("EASYPOST_API_KEY")

	batch, _ := client.CreateBatchScanForms("batch_...", "pdf")

	fmt.Println(batch)
}
